sdk/core/azure-core-http-netty/README.md

```diff
@@ -22,6 +22,30 @@ Azure Core Netty HTTP client is a plugin for the azure-core HTTP client API.
 
 ## Examples
 
+The following sections provide several code snippets covering some of the most common client configuration scenarios.
+
+- [Create a Simple Client](#create-a-simple-client)
+- [Create a Client with Proxy](#create-a-client-with-proxy)
+
+### Create a Simple Client
+
+Create the simplest Netty Http client.
```

Comment:
Would it be useful to elaborate a bit on what a simple Netty client would do? For e.g. what are the default values it would be configured with if someone creates this client.